### PR TITLE
use micromamba to install `fitsverify`

### PR DESCRIPTION
**Pull request by zacharyburnett**
we can install `fitsverify` in a Conda environment with the `setup-micromamba` action; this should dramatically decrease the time to get `fitsverify`; however, it does not build it from source, instead gets the latest `conda-forge` build

### CODE REVIEW

**Comment by zacharyburnett** [comment]
it looks like the last two failing tests in the current `pytest-migration` branch are in checksums in `test_certify.py`:
https://github.com/spacetelescope/crds/actions/runs/6419119353/job/17428298626#step:7:29
```
test/certify/test_certify.py .FF........................................ [ 35%]
```

**Comment by zacharyburnett** [comment]
```
2023-10-09 19:34:25,262 - stpipe - WARNING -  External fitsverify program is enabled but not found on PATH.
```

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -61,28 +65,22 @@ jobs:
...
-          pip install git+https://github.com/spacetelescope/jwst
-          pip uninstall --yes crds
-          ./install
-          pip install .[submission,test,docs,synphot]
+          environment-name: crds-testing
+          create-args: >-
+            python=3
+            asdf
+            astropy
+            filelock
```
Comment (zacharyburnett) [inline .github/workflows/ci.yml:75]:
```suggestion
            filelock
            fitsverify
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -61,28 +65,23 @@ jobs:
...
-      - name: Install Python dependencies
-        run: |
-          pip install roman-datamodels
-          pip install git+https://github.com/spacetelescope/jwst
-          pip uninstall --yes crds
-          ./install
-          pip install .[submission,test,docs,synphot]
+          environment-name: crds-testing
+          create-args: >-
+            python=3
```
Comment (zacharyburnett) [inline .github/workflows/ci.yml:72]:
```suggestion
            python=3.11
```

**Review by alphasentaurii — APPROVED** [review]

**Review by hbrown-ST — APPROVED** [review]

**Pull request merged**